storage/elasticsearch/build.sbt

```diff
@@ -25,10 +25,10 @@ libraryDependencies ++= Seq(
   "org.apache.predictionio" %% "apache-predictionio-core" % version.value % "provided",
   "org.apache.spark"        %% "spark-core"               % sparkVersion.value % "provided",
   "org.elasticsearch.client" % "elasticsearch-rest-client" % elasticsearchVersion.value,
-  "org.elasticsearch"       %% "elasticsearch-spark-20" % elasticsearchVersion.value
+  "org.elasticsearch"       % "elasticsearch-spark-20_2.11" % elasticsearchVersion.value
```

Comment:
Does this run on Spark 2.4.3 (Scala 2.12)?

Reply:
As you pointed out, tests currently only run under 2.11, so I'm not sure. I wanted to ensure the 2.11 build was working first. That's the only 2.11 dep that doesn't have a 2.12 equivalent. 
I'll fix .travis.yml now to test under 2.12.

Reply:
I did test locally with 2.12 and elasticsearch, did not encounter any issues.

Reply:
Ahh, too good to be true, of course.
It can't work with the 2.11 version in 2.12, so that is a blocker.

Reply:
https://github.com/elastic/elasticsearch-hadoop/issues/1224

Reply:
PR is here, so we can pick this back up once they've merged and published:

https://github.com/elastic/elasticsearch-hadoop/pull/1308